src/main/java/rotodo/tasklist/TaskList.java

```diff
@@ -0,0 +1,187 @@
+package rotodo.tasklist;
+
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.IntStream;
+
+import rotodo.exception.InvalidInputException;
+
+/**
+ * This class implements the TaskList. The list can
+ * be interacted and modified through provided functions.
+ *
+ * @author Ng Kay Hian
+ * @version CS2103T AY24/25 Semester 1
+ */
+public class TaskList {
+    private boolean nextStatus;
+
+    /**
+     * List of tasks.
+     */
+    private List<Task> list;
```

Comment:
Perhaps a better variable name here would be `tasks`?

Reply:
agreed ^